apps/src/Sound.js

```diff
@@ -340,10 +371,17 @@ Sound.prototype.preloadViaWebAudio = function (filename, onPreloadedCallback) {
   request.responseType = 'arraybuffer';
   var self = this;
   request.onload = function () {
-    self.audioContext.decodeAudioData(request.response, function (buffer) {
-      onPreloadedCallback(buffer);
-      self.onSoundLoaded();
```

Comment:
`onSoundLoaded()` is called in 2 places. One for webaudio (here) and one for HTML5 audio (line 320 above). I think you may need to do something similar to handle failures in the HTML5 audio case. If we have trouble loading HTML5 audio, we'll never get the `canplaythrough` event, but hopefully we will get something else that you can use to trigger the new `handleLoadFailed()`. This code path is executed on IE today, but you can probably force it to execute on Chrome for testing purposes.

Reply:
Thanks Chris, good catch. I've added a call to handleLoadFailed() in the html5 case. Because the http status code is not available to us in that case, we'll be sure to get out of the gray-run-button state but we won't try to refetch cookies. This means IE11 users who keep a dance level open for more than 4 hours will have to reload the page in response to their song and app not running. 

Alternately, I could pass a special "unknown" status code in the html5 case and the client could retry based on this status code. This would mean we'd refetch cookies and song even if a different error besides 403 occurs. This seems a bit hacky but could be worth it. IE11 is 1.5% of users. During HoC, that could be > 300K users but it's hard to say how many of them will hit the 4 hour mark without reloading the page.